### ISSUE TITLE
feature request: request for some accessibility impruvments

### ISSUE THREAD

**Issue by ilyapashuk** [termux/termux-app#1586]
**Feature description**

Termux is most screen reader accessible terminal emulator that I seen.
In most popular android terminals screenreaders was even unable get a terminal view text, so that apps was fully unaccessible.
In termux, you provided a good accessibility because screenreaders are able to get view's text and notified about all it's changes.
but android screenreaders knows nothing about cli consoles and often handles sutch events incorrectly.
for exampl, on text change, they re reads full text, not only changed part, or reads nothing. Also, it's common practice that when some letter typed on the keyboard, physical or touch, screeners read that character and, when space are pressed, they read full word to make sure that it is correct, but in termux nothing of this works.

to make this app more accessible, you should add some changes to it's behavior:
view should not generate default TextChanged event, but it should use an AnnounceForAccessibility method to announce only new or changed textparts.
when a letter are typed, it should send that letter to AnnounceForAccessibility too.
when del key are typed, it should speak something like "%l deleted" where %l is deleted letter.
and on any cursor movement it should speak new cursor position (new cursored line in up-down movement and new word on left-right movement)

also, I want to test that changes before publication as an accessibility user, but I need a ready apk because at this time have not a working build instrumentation. And also it's version code should be equal with current release code because I have a lot of data in termux env and can't reinstall it to downgrade. The same sing for signing.
or it should be as different package name, failsafe mode will be enough for first time.

thank you
